genshi/path.py

```diff
@@ -537,7 +537,7 @@ def __init__(self, text, filename=None, lineno=-1):
                     self.strategies.append(strategy_class(path))
                     break
             else:
-                raise NotImplemented('No strategy found for path')
+                raise NotImplementedError('No strategy found for path')
```

Comment:
Doh. I wonder how long this silly mistake has been here. Thanks for catching it.